### DISCUSSION TITLE
This may be a dumb question but does it need to be installed at all?

### DISCUSSION THREAD

**Discussion by ghost** [eoyilmaz/displaycal-py3#363] [Q&A]
Can't I just run it from the extracted folder via "python3 ./DisplayCAL.pyw" in terminal? It seems to work but I'm wondering if I'm missing anything...sorry if this is a really dumb question.

**Comment by p5k369** [comment]
When people talk about installing DisplayCal, they usually mean installing the package in a virtual environment. This is not necessary to run displaycal.
